test/vars/UtilSpec.groovy

```diff
@@ -555,4 +555,13 @@ class UtilSpec extends JenkinsPipelineSpecification {
         then:
         1 * getPipelineMock("error").call('Version 0.12.0.1 is not in the required format X.Y.Z')
     }
+
+    def "[util.groovy] generateHash"() {
+        when:
+        def hash9 = groovyScript.generateHash(9)
+        def hash1000 = groovyScript.generateHash(1000)
+        then:
+        hash9.length() == 9
+        hash1000.length() == 1000
+    }
```

Comment:
I would suggest two different tests here, one for 9 and another one for 1000.

Reply:
updated